src/viewer.js

```diff
@@ -3415,6 +3417,43 @@ function onCanvasPinch( event ) {
     }
 }
 
+function onCanvasFocus( event ) {
+
+    /**
+     * Raised when a pointer focus on the {@link OpenSeadragon.Viewer#canvas} element.
+     *
+     * @event canvas-focus
+     * @memberof OpenSeadragon.Viewer
+     * @type {object}
+     * @property {OpenSeadragon.Viewer} eventSource - A reference to the Viewer which raised this event.
+     * @property {OpenSeadragon.MouseTracker} tracker - A reference to the MouseTracker which originated this event.
+     * @property {Object} originalEvent - The original DOM event.
+     * @property {?Object} userData - Arbitrary subscriber-defined object.
+     */
+    this.raiseEvent( 'canvas-focus', {
+        tracker: event.eventSource,
+        originalEvent: event.originalEvent
+    });
+}
+
+function onCanvasBlur( event ) {
+    /**
+     * Raised when a pointer blur on the {@link OpenSeadragon.Viewer#canvas} element.
```

Comment:
I think a better phrasing would be `Raised when the {@link OpenSeadragon.Viewer#canvas} element loses keyboard focus.`